src/parser/jobs/smn/modules/Cooldowns.js

```diff
@@ -3,16 +3,30 @@ import ACTIONS from 'data/ACTIONS'
 
 export default class Cooldowns extends CoreCooldowns {
 	static cooldownOrder = [
-		ACTIONS.AETHERFLOW.id,
+		ACTIONS.ENERGY_DRAIN.id,
+		ACTIONS.ENERGY_SIPHON.id,
 		ACTIONS.FESTER.id,
 		ACTIONS.PAINFLARE.id,
-		ACTIONS.ENERGY_DRAIN.id,
 		ACTIONS.BANE.id,
 		ACTIONS.DREADWYRM_TRANCE.id,
 		ACTIONS.DEATHFLARE.id,
 		ACTIONS.SUMMON_BAHAMUT.id,
 		ACTIONS.ENKINDLE_BAHAMUT.id,
-		ACTIONS.ROUSE.id,
-		ACTIONS.ENKINDLE.id,
+		ACTIONS.FIREBIRD_TRANCE.id,
```

Comment:
Similar story down here; stuff that's mutually exclusive can be grouped to semantically show their logic.